solr_conf/conf/solrconfig.xml

```diff
@@ -11,10 +11,10 @@
   <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.StandardDirectoryFactory}"/>
 
   <!-- solr lib dirs -->
```

Comment:
Some part of it is being used. I added https://github.com/ualbertalib/HydraNorth/blob/master/solr_conf/conf/schema.xml#L356 to work with changes I made to solrizer, and it works in dev/test, so jetty's picking that up in those environments.

In Production I think we're using Solr Cloud (?? documentation would once again be a huge benefit for us all here) but there must be some way these custom Hydra schemas get picked up by it...